app/views/clients/rollup/_client_notes.html.haml

```diff
@@ -0,0 +1,10 @@
+- @client.notes.order(created_at: :desc).each do |note|
+  .well.client-notes
+    %h4= note.type_name
+    %light= note.note
```

Comment:
I think you mean `.light`, but I'm not sure that's defined, and I didn't intend for it to be on the note, but on the attribution.  This should probably just be %p